roles/openshift_get_core_cluster/tasks/main.yml

```diff
@@ -28,11 +28,28 @@
   register: networks
   changed_when: false
 
+- name: Create floating ips for each core cluster VM
+    shell: "{{ openstack }} floating ip create {{ openstack_public_network_name }} --format value -c floating_ip_address"
+    register: floating_addresses
+    with_sequence: "{{ networks['stdout_lines']|length }}"
```

Comment:
Looks like items below name are indented extra

Reply:
Yes it is, let me fix that.